content/post/frontend_talks_2017.md

```diff
@@ -0,0 +1,142 @@
+---
+author: erizocosmico
+date: 2017-07-10
+title: "source{d} tech talks, frontend series"
+image: /post/frontend-talks/header-pic.jpg
+description: "Once every few months, source{d} organizes small conferences around a very specific topic. On June, 24th the topic was frontend and the talks were hosted in our Madrid offices. Almost 50 people joined us for a day full of things to learn about frontend technologies."
+categories: ["technical", "frontend"]
+---
+
+![Schedule with our logo](/post/frontend-talks/header-full.jpg)
+
+Once every few months, source{d} organizes small conferences around a very specific topic. On June, 24th the topic was frontend and the talks were hosted in our Madrid offices. Almost 50 people joined us for a day full of things to learn about frontend technologies.
+
+The topics of the talks were very diverse, from animations with Ember to multi-threaded web to React styiling to accessibility. Since frontend is such a wide area in programming, we aimed to provide talks from a lot of different topics, which we thing we accomplished.
+
+## Conferences are no fun on an empty stomach
+
+First things first, we started the day with a nice welcome breakfast for all the attendees and speakers in which people could talk and get to know each other as well as the speakers before their talks.
+
+## Opening speech
+
+After everyone was in their seat, [Margarida García](https://twitter.com/margaridagsl) gave the opening talk and welcomed all the people in the room. She talked a bit about what we do here at source{d} and explained what the source{d} tech talks was about and why the community was so important to us.
+
+## Talks
+
+The day was split in two sessions: the morning and the afternoon sessions. The morning session had 3 main talks and the afternoon session only two, having the morning session a lightning talk at the end of it and another lightning talk at the start of the afternoon session. Due to a last-minute cancellation of a speaker the lightning talk from the afternoon session was removed.
+Between sessions there was a lunch break and plenty of coffee breaks during each of the sessions.
+All main talks lasted 45 minutes, 30 for the talk and 15 for questions and answers while the lightning talks lasted for 15 minutes and there was no time for questions and answers.
+
+## Morning session
+
+### The multithreaded web: a tale of workers, _Ramón Guijarro_
+
+In the first morning talk, [Ramón Guijarro](https://twitter.com/soyguijarro), frontend developer at Tuenti, talked about how to develop multi-threaded applications in the web using workers.
+
+Ramón started his talk mentioning how the release of the iPhone and afterwards the App Store started a change in user expectations. So we've had to move from web pages to web apps to meet those user expectations on the web.
```

Comment:
Regarding **with**:

He started doing math. -- Translation: he began to do study math and we do not know what happened next. He just did it. Dot.
He started with doing math. -- Translation: he decided to do his math first and then he was likely to do smth else. This implies the sequence of actions.
Example: https://www.reddit.com/r/socialmedia/comments/5jx8bg/where_to_start_with_doing_facebook_ads/ The guys asks what should he do in the first place to reach his goal.

Regarding iPhone and App Store:

https://en.wikipedia.org/wiki/IPhone
https://en.wikipedia.org/wiki/App_Store_(iOS)

This is actually funny: they use "the iPhone" but "App Store". Crazy Americans. So we should leave the iPhone but remove "the" before App Store.

Regarding started to change: yep, to start (to do|doing) smth https://english.stackexchange.com/questions/133676/which-is-correct-started-to-change-or-is-starting-to-change